Outlines/Bonding Curves.md

```diff
@@ -0,0 +1,15 @@
+Bonding Curves
+- Bonding Curves are pricing curves that mediate the supply of a token. (include longer definition & elaboration)
```

Comment:
@calchulus Yes, and especially expand on why these might be useful to someone, what problem are they trying to solve?

Reply:
Yep sounds awesome to me